assets/scss/projects.scss

```diff
@@ -28,6 +28,48 @@ html.modal-open, html.modal-open body {
     }
   }
 
+  .protocols-summary {
+    th:nth-child(7),
+    td:nth-child(7) {
+      padding-left: 20px;
+    }
+
+    tr.expandable {
+      cursor: pointer;
+
+      &.hover, &.expanded {
+        background-color: white;
+      }
+
+      &.expanded {
+        td.is-last-row {
```

Comment:
Why not a `:last-child`?

Reply:
doesn't work - we want it for all cells in the last row, which can be the first row or the nth row. Because rowspan

Reply:
I'm still not sure I get this.

`tr:last-child td` would match all the cells in the last row, right?

Reply:
yep, but not the cells in the first row, which are actually the last row cos rolspan

Reply:
also it's not the last child, there are multiple <tr>s per protocol, so it's the last species in a a protocol, I tried with css and couldn't get anything close without a class.

Reply:
🤯

Reply:
#shipit then.

Reply:
<img width="1266" alt="Screen Shot 2020-05-06 at 11 42 59" src="https://user-images.githubusercontent.com/3604142/81168524-109c5e80-8f8f-11ea-8d5b-b79fb59d69a0.png">
VS
<img width="1237" alt="Screen Shot 2020-05-06 at 11 43 58" src="https://user-images.githubusercontent.com/3604142/81168530-12662200-8f8f-11ea-942b-028554ddb490.png">